scripts/install_dependencies.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+CURRENT_DIR=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)
+
+unameOut="$(uname -s)"
+case "${unameOut}" in
+  Linux*)
+    machine=Linux
+    ;;
+  Darwin*)
+    machine=macOS
+    ;;
+  *)
+    echo "This script isn't supported on your OS '${unameOut}'. Please, use Linux or macOS"
+    exit 1
+esac
+
+if [ "${machine}" == "Linux" ];
+then
+  "${CURRENT_DIR}/install_deps_ubuntu.sh" "$@"
```

Comment:
I think that it'd be better to add additional check before this line that tries to find `apt` in the system. Then if apt wasn't found we'll show the warning about that Debian-based distros are the only officially supported and you'll have to install the following packages by yourself.